src/dashboard/withdraw/RequestList.tsx

```diff
@@ -245,44 +245,20 @@ function RequestList( {
                 isLoading={ loading }
             />
 
-            <Modal
-                className="max-w-2xl dokan-withdraw-style-reset dokan-layout"
+            <DokanModal
                 isOpen={ isOpen }
+                namespace="cancel-request-confirmation"
+                onConfirm={ () => canclePendingRequest() }
                 onClose={ () => setIsOpen( false ) }
-                showXButton={ false }
-            >
-                <Modal.Title className="border-b">
-                    { __( 'Confirm', 'dokan' ) }
-                </Modal.Title>
-                <Modal.Content className="">
-                    <p className="text-gray-600">
-                        { __(
-                            'Are you sure, you want to cancel this request ?',
-                            'dokan'
-                        ) }
-                    </p>
-                </Modal.Content>
-                <Modal.Footer className="border-t">
-                    <div className="flex flex-row gap-3">
-                        <Button
-                            color="secondary"
-                            className="bg-gray-50 hover:bg-gray-100"
-                            onClick={ () => setIsOpen( false ) }
-                            disabled={ withdrawHook.isLoading }
-                            label={ __( 'No', 'dokan' ) }
-                        />
-
-                        <Button
-                            color="secondary"
-                            className="bg-gray-50 hover:bg-gray-100"
-                            onClick={ canclePendingRequest }
-                            disabled={ withdrawHook.isLoading }
-                            loading={ withdrawHook.isLoading }
-                            label={ __( 'Yes', 'dokan' ) }
-                        />
-                    </div>
-                </Modal.Footer>
-            </Modal>
+                confirmationTitle={ __( 'Are you sure', 'dokan-lite' ) }
+                confirmationDescription={ __(
+                    'You want to cancel this request?',
```

Comment:
Pls make the sentence to interrogative  from`  'You want to cancel this request?'` to`'Do You want to cancel this request?'`